service-discovery/static-list/src/main/java/io/smallrye/stork/servicediscovery/staticlist/StaticListServiceRegistrar.java

```diff
@@ -0,0 +1,29 @@
+package io.smallrye.stork.servicediscovery.staticlist;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.smallrye.mutiny.Uni;
+import io.smallrye.stork.api.Metadata;
+import io.smallrye.stork.api.ServiceRegistrar;
+import io.smallrye.stork.spi.StorkInfrastructure;
+
+public class StaticListServiceRegistrar implements ServiceRegistrar<Metadata.DefaultMetadataKey> {
+    private static final Logger log = LoggerFactory.getLogger(StaticListServiceRegistrar.class);
+    private final StaticRegistrarConfiguration config;
+
+    public StaticListServiceRegistrar(StaticRegistrarConfiguration config, String serviceName,
+            StorkInfrastructure infrastructure) {
+        this.config = config;
+    }
+
+    @Override
+    public Uni<Void> registerServiceInstance(String serviceName, Metadata<Metadata.DefaultMetadataKey> metadata,
+            String ipAddress,
```

Comment:
should we add the scheme and path?

Reply:
Yes why not but given that the original API doesn't contain it, do you mind to do it in another PR?